### PR TITLE
Revised SDL-0266 New vehicle data GearStatus

### PR DESCRIPTION
**Pull request by atiwari9**
This proposal is to add new vehicle data `GearStatus` to know drive mode and actual gear the transmission has selected.

### CODE REVIEW

**Comment by atiwari9** [comment]
@theresalech  - Ready for review.

File: proposals/0266-New-vehicle-data-GearStatus.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ We need to add `GearStatus` for `GetVehicleData`, `SubscribeVehicleData`, `Unsub
 
 ```xml	
 <struct name="GearStatus" since="X.x">
-	<param name="selectedGear" type="PRNDL" mandatory="false">
-		<description>Tells the gear drive mode i.e. PARK, DRIVE, SPORT etc</description>
+	<param name="userSelectedGear" type="PRNDL" mandatory="false">
+		<description>Gear position selected by the user i.e. Park, Drive, Reverse</description>
 	</param>
 	<param name="actualGear" type="PRNDL" mandatory="false">
 		<description>Tells the actual gear selected by transmission</description>
```
Comment (theresalech) [inline proposals/0266-New-vehicle-data-GearStatus.md]: Based on https://github.com/smartdevicelink/sdl_evolution/issues/881#issuecomment-573838656, this should be
```suggestion
		<description>Actual Gear in use by the transmission</description>
```

File: proposals/0266-New-vehicle-data-GearStatus.md [hunk 2]
```diff
@@ -173,8 +173,8 @@ We need to add `GearStatus` for `GetVehicleData`, `SubscribeVehicleData`, `Unsub
 
 ```xml	
 <struct name="GearStatus">
-	<param name="selectedGear" type="Common.PRNDL" mandatory="false">
-		<description>Tells the gear drive mode i.e. PARK, DRIVE, SPORT etc</description>
+	<param name="userSelectedGear" type="Common.PRNDL" mandatory="false">
+		<description>Gear position selected by the user i.e. Park, Drive, Reverse</description>
 	</param>
 	<param name="actualGear" type="Common.PRNDL" mandatory="false">
 		<description>Tells the actual gear selected by transmission</description>
```
Comment (theresalech) [inline proposals/0266-New-vehicle-data-GearStatus.md]: Based on https://github.com/smartdevicelink/sdl_evolution/issues/881#issuecomment-573838656, this should be
```suggestion
		<description>Actual Gear in use by the transmission</description>
```

File: proposals/0266-New-vehicle-data-GearStatus.md (hunk 1 above)
Comment (atiwari9) [reply proposals/0266-New-vehicle-data-GearStatus.md]: Done

File: proposals/0266-New-vehicle-data-GearStatus.md (hunk 2 above)
Comment (atiwari9) [reply proposals/0266-New-vehicle-data-GearStatus.md]: Done

**Comment by atiwari9** [comment]
@theresalech  - I didn't realize that Jack had updated description for `actualGear` as well. I have updated that as well now.

**Pull request merged**